ios/BetterSocial/Info.plist

```diff
@@ -45,26 +29,40 @@
 			<string>org.bettersocial</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>me.bettersocial.org</string>
+				<string>helio</string>
 			</array>
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>FirebaseDynamicLinksCustomDomains</key>
+	<array>
+		<string>https://link.bettersocial.org</string>
+	</array>
+	<key>LSApplicationQueriesSchemes</key>
+	<array>
+		<string>mailto</string>
+		<string>message</string>
+		<string>readdle-spark</string>
+		<string>airmail</string>
+		<string>ms-outlook</string>
+		<string>googlegmail</string>
+		<string>inbox-gmail</string>
+		<string>ymail</string>
+		<string>superhuman</string>
+		<string>yandexmail</string>
+		<string>fastmail</string>
+		<string>protonmail</string>
+		<string>szn-email</string>
+	</array>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 		<key>NSExceptionDomains</key>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The URL scheme has been changed from `me.bettersocial.org` to `helio`. This change is fine as long as it aligns with the intended deep linking strategy. However, please ensure that all parts of your application and any external systems that might rely on this URL scheme are updated accordingly to prevent broken links.

Also, you've added Firebase Dynamic Links custom domain `https://link.bettersocial.org`. Make sure that this domain is correctly configured in your Firebase console and that the necessary DNS records have been set up for this domain.

Lastly, the `NSExceptionDomains` key is now empty. If you had specific domains that needed exceptions for App Transport Security (ATS), they should be listed here. If not, then this change is fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->